web/themes/custom/westy/templates/content/node--alert--featured.html.twig

```diff
@@ -0,0 +1,38 @@
+{# Set Alert Type #}
+{% if node.field_severity.value == 20 %}
+	{% set alertType = 'warning' %}
```

Comment:
Tab spacing/indentation seems to be all over the place in this file... might as well correct it with Josh's other requested changes.

here's it's about a 8 space tab.